test/controllers/products_controller_test.rb

```diff
@@ -0,0 +1,23 @@
+require "test_helper"
+
+class ProductsControllerTest < ActionDispatch::IntegrationTest
+  test "should get index" do
+    get products_index_url
+    assert_response :success
+  end
+
+  test "should get show" do
```

Comment:
you can update the test files to use single quotes instead of double quotes where you don't have interpolation